### PR TITLE
Add otp settings model

### LINKED ISSUE #73: Update the user model to include the new OTP settings model
The user settings model should include the new OTP settings.

### PR DESCRIPTION
**Pull request by SandroMachado**
Closes #73.

### CODE REVIEW

File: src/app/src/main/java/com/uphold/uphold_android_sdk/model/user/Settings.java [hunk 1]
```diff
@@ -22,14 +23,16 @@
      * @param hasOtpEnabled A boolean indicating if the user has the OTP enable.
      * @param intl The user internationalization settings.
      * @param theme The user theme.
+     * @param otp The user OTP settings.
      */
 
-    public Settings(String currency, Boolean hasNewsSubscription, Boolean hasOtpEnabled, InternationalizationUserSettings intl, String theme) {
+    public Settings(String currency, Boolean hasNewsSubscription, Boolean hasOtpEnabled, InternationalizationUserSettings intl, String theme, OTP otp) {
```
Comment (diogoguimaraes) [inline src/app/src/main/java/com/uphold/uphold_android_sdk/model/user/Settings.java]: Please sort the `top` parameter and the documentation.

**Comment by diogoguimaraes** [comment]
LGTM.

File: src/app/src/androidTest/java/com/uphold/uphold_android_sdk/test/integration/client/UpholdClientTest.java [hunk 2]
```diff
@@ -287,19 +286,37 @@ public void getUserShouldReturnTheUser() throws Exception {
               "\"currency\": \"USD\"," +
               "\"hasNewsSubscription\": \"true\"," +
               "\"intl\": {" +
-                  "\"language\": {" +
-                      "\"locale\": \"en-US\"" +
+                "\"language\": {" +
+                    "\"locale\": \"en-US\"" +
+                "}," +
+                "\"dateTimeFormat\": {" +
+                    "\"locale\": \"en-US\"" +
```
Comment (rafaelveloso) [inline src/app/src/androidTest/java/com/uphold/uphold_android_sdk/test/integration/client/UpholdClientTest.java]: Indentation is off.

File: src/app/src/main/java/com/uphold/uphold_android_sdk/model/user/OTP.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
+package com.uphold.uphold_android_sdk.model.user;
+
+import java.io.Serializable;
+
+/**
+ * OTP settings model.
+ */
+
+public class OTP implements Serializable {
```
Comment (ruipenso) [inline src/app/src/main/java/com/uphold/uphold_android_sdk/model/user/OTP.java]: `Otp`

File: src/app/src/androidTest/java/com/uphold/uphold_android_sdk/test/util/Fixtures.java [hunk 4]
```diff
@@ -15,9 +14,16 @@
 import com.uphold.uphold_android_sdk.model.transaction.TransactionDenominationRequest;
 import com.uphold.uphold_android_sdk.model.transaction.TransactionRequest;
 import com.uphold.uphold_android_sdk.model.user.ContactRequest;
-import com.uphold.uphold_android_sdk.model.user.InternationalizationUserSetting;
-import com.uphold.uphold_android_sdk.model.user.InternationalizationUserSettings;
+import com.uphold.uphold_android_sdk.model.user.settings.internationalizationusersettings.InternationalizationUserSetting;
+import com.uphold.uphold_android_sdk.model.user.settings.InternationalizationUserSettings;
```
Comment (ruipenso) [inline src/app/src/androidTest/java/com/uphold/uphold_android_sdk/test/util/Fixtures.java]: Rename `InternationalizationUserSettings` to `Intl` in another PR.

**Pull request merged**